docs/blog/2019-09-11-gazette-september/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "Gatsby Gazette: The September Installment"
+date: 2019-09-09
+author: Sidhartha Chatterjee
+excerpt: "It's September and we're back with a brand new Gatsby Gazette, this time featuring a theme of the month, an RFC to localise documentation and Gatsby Days"
+tags:
+  - gazette
+---
+
+How was August for all of you? It's been rainy here in Mumbai. Like [_really_](https://qz.com/india/1703369/photos-of-mumbai-rains-and-floods-amid-indian-monsoons/) rainy. But let's not digress cause it's time for the Gatsby Gazette!
+
+This month's Gazette features our work on materialization, a healthier Gatsby repo with improved community engagement, some accessibility wins, better webpack chunking, a better offline experience in Gatsby, a brand new theme of the month, and finally, Gatsby Days.
+
+Let's get this party started.
+
+##
+
+## Team Highlights
+
+The Gatsby team has had a busy month. We've made some exciting improvements.
+
+### Accessibility
+
+[Marcy Sutton](https://twitter.com/marcysutton) from Gatsby's Learning team has been doing some incredible work in making Gatsby more accessible by default.
+
+A big part of making the web more accessible is teaching us how to write more accessible apps! Back in June, Marcy conducted some user testing sessions for accessibility research. She wrote about it in her post, [What we learned from user testing of accessible client-side routing techniques with Fable Tech Labs](https://www.gatsbyjs.org/blog/2019-07-11-user-testing-accessible-client-routing/) which I consider canonical and _required_ reading.
+
+Marcy also presented a full-length workshop on [Accessibility in JavaScript Applications for Frontend Masters](https://frontendmasters.com/workshops/javascript-accessibility/), including some exclusive tidbits about building accessible Gatsby sites. In case you haven't read her post or seen her workshop already, please _do_ to learn how to make your web apps more accessible.
+
+### GraphQL
+
+We've made a bunch of improvements to Gatsby's GraphQL layer recently. Here is a quick summary:
+
+- We now support abstract data interfaces. This means that we can now build Gatsby sites (or themes) that support the same query for multiple data sources. [Mikhail](https://twitter.com/freiksenet) from the core team talks more about this on [Learn With Jason](https://www.youtube.com/watch?v=ALspNtrOqDk)
+- Materialisation
```

Comment:
Something along these lines might work here?

_This work included a major refactoring in Gatsby’s data layer. It restores support for a database storage backend within Gatsby, which speeds up builds for larger sites._

Reply:
Super useful! Thank you. I was wondering how to describe @freiksenet's incredible work here succinctly